### PR TITLE
What's New - TM:PE 11.6.6.0

### PR DESCRIPTION
**Pull request by aubergine10**
- **[Meta] This release adds a new language, updates translations, numerous fixes and improves lane routing**
- [New] Lightweight highway merging rules #1542 (kianzarrin)
- [New] Allow buses and taxi to use Old Town districts (optional) #1618 (krzychi124)
- [New] Notify the user in case of errors while loading savegame data #1617 (krzychu124)
- [New] Vietnamese translation #1551 (DucAnhLam)
- [Fixed] Lane connections save/restore issues (kianzarrin)
- [Fixed] Broken detection of hovered element #1594 (krzychu124)
- [Fixed] Incorrectly rendered lane connections #1605 #1604 (krzychu124)
- [Fixed] DLS incorrectly calculate 'max unsafe speed difference' #1581 (krzychu124)
- [Fixed] Redundant transitions at intersections with mixed rail/road vehicle lanes #1565 (kianzarrin)
- [Fixed] Trams/Trains get blocked by their own trailer when altering junction settings #1574 #1572 (kianzarrin)
- [Fixed] Improved pedestrian crossing gap calculations (very wide roads, with 10+ lanes) #1610 (kianzarrin)
- [Fixed] Hot-reload for API #1602 (kianzarrin)
- [Updated] Separate road and track lane routing #1550 #1546 #1545 #354 (kianzarrin)
- [Updated] Translations for multiple languages #1551 #1344 (Nguyễn Tài Đức, OldEj, AduitSSH, Chamëleon, 문주원, krzychu124, shg166, John Lok Ho, Márcio Saeger, DucAnhLam, DNSE, Arne Peirs, Neoone, Fatih YILDIRIM, Zeldslayer, GiorgioHerbie, krzychu124)

### CODE REVIEW

**Review by kvakvs — APPROVED** [review]
Nice

**Comment by krzychu124** [comment]
Cool, I think we can fit #1542, #1565. #1574 and #1526 (#1550 needs to be merged first) - so pretty much all that are already waiting for approval or tests. 

I'll cherry-pick translation update into last released `Test` version (hotfix-1) and if will work without issues it will go to `Test` and `Stable` once @kianzarrin confirm he is ready to update `AN` plus other mods waiting for API changes from `Test`. Current `master` cannot go to `Test` before pushing the `Test` to `Stable`. I hope it's the last time with such mess in release process...

**Review by kianzarrin — APPROVED** [review]

**Comment by krzychu124** [comment]
**Important note:** nothing between tag `11.6.5.2` and your locale PR #1551 was not released to the Test and should be added to this changelog (PR's listed below) 😕 
#1545 #1546 #1556

**Review by kvakvs — APPROVED** [review]

**Review by krzychu124 — APPROVED** [review]

**Pull request merged**
